job/genericjob/keras-multiworker/README.md

```diff
@@ -0,0 +1,13 @@
+# 使用 GenericJob 实现 Keras 模型的多工作器同步训练
+
+本示例演示如何使用 GeneribJob 对 Keras 模型进行多工作器同步训练（采用 [`tf.distribute.MultiWorkerMirroredStrategy`](https://www.tensorflow.org/api_docs/python/tf/distribute/MultiWorkerMirroredStrategy) 分布式策略）。本示例所使用的脚本文件和示例[使用 TensorFlowTrainingJob 进行多工作器同步训练](../../tensorflowtrainingjob/multiworker/)完全相同，GenericJob 的这一 YAML 配置实际上就是 TensorFlowTrainingJob 的具体实现。
```

Comment:
please briefly explain the differences in using GenericJob vs. TensorFlowTrainingJob

Reply:
updated